group_vars/production

```diff
@@ -5,6 +5,7 @@ wordpress_sites:
     site_hosts:
       - example.com
       - 192.168.50.5
+    local_path: '../example.dev' # path targeting local project directory (relative to root/Vagrantfile)
```

Comment:
The README [installation section](https://github.com/roots/bedrock-ansible/blob/3f5c44d097ef0c71d81bee705b37042e65178aba/README.md#installation) implies naming your Bedrock directory `example.com`, so maybe this could avoid potential confusion if it were`.com` instead of `.dev`. Same for other `group_vars` files, especially `group_vars/development` because it is the one that new users will encounter first.